@here/harp-mapview/test/MapViewTest.ts

```diff
@@ -306,4 +281,46 @@ describe("MapView", function() {
             }
         }
     });
+
+    it("updates background tiling scheme", async function() {
+        this.timeout(500);
+        let setupFrameCallback: FrameRequestCallback = () => {};
+
+        let customMapView: MapView;
```

Comment:
Please reuse suite `mapView` variable, as it ensures `mapView` is destroyed regardless of test outcome.

Test case in this state will leak live `MapView` instance which can do some async stuff after the test case and fail random test cases later.

Reply:
done.